assets/dev/js/editor/commands/close.js

```diff
@@ -14,8 +14,8 @@ export class Close extends CommandBase {
 			return jQuery.Deferred().resolve();
 		}
 
-		// TODO: Move to an hook.
-		if ( ! mode && ( document.editor.isChanged || document.isDraft() ) ) {
+		// TODO: Move to an hook ( UI Hook not under tests ).
+		if ( ! elementorCommonConfig.isTesting && ! mode && ( document.editor.isChanged || document.isDraft() ) ) {
```

Comment:
Remove `elementorCommonConfig.isTesting`.

Reply:
Simple removing of 'elementorCommonConfig.isTesting' will trigger test error, the solution is move it to an hook, i will do  and create new UI hook for handling this.

Reply:
handled.